packages/bits/demo/src/components/demo/popover/popover-prevent-close-on-click/popover-prevent-close-on-click.example.component.html

```diff
@@ -14,7 +14,10 @@
         <nui-checkbox id="nui-demo-checkbox-in-popover" i18n>Hi, check me!</nui-checkbox>
         <nui-combobox id="nui-demo-popover-combobox"
                       [itemsSource]="dataset.items"
-                      placeholder="Select item" i18n-placeholder></nui-combobox>
+                      placeholder="Select item combobox" i18n-placeholder></nui-combobox>
+        <nui-combobox-v2 id="nui-demo-popover-comboboxV2" placeholder="Select item comboboxV2" i18n-placeholder>
+            <nui-select-v2-option *ngFor="let item of dataset.items" [value]="item">{{ item }}</nui-select-v2-option>
+        </nui-combobox-v2>
```

Comment:
Hi, Pavlo. Thanks for contributing!

I think it would be best to not add this combobox-v2 instance to the example since it doesn't add to the comprehension of how to use popover. We try to keep our examples in the service of understanding component usage as much as possible and try not to include anything that might distract from that.

Reply:
Oh, yep, got it. I will remove it from here. I've seen you have the ticket to remove old combobox hope this will be covered.

Reply:
Good point. We'll make sure to replace this instance of nui-combobox with nui-combobox-v2 when the time comes. Thanks again!